src/core/StateSerializationHelper.cpp

```diff
@@ -114,15 +114,24 @@ bool StateSerializationHelper::load( const QString& filename )
                       filename.toStdString().c_str(), e.what( ));
             return false;
         }
+
         ifs.close();
     }
 
-    ContentWindowPtrs contentWindows = state.getContentWindows();
     if( state.getVersion() < FIRST_PIXEL_COORDINATES_FILE_VERSION )
-        scaleToDisplayGroup( contentWindows );
-    validate( contentWindows );
+        scaleToDisplayGroup( state.getContentWindows());
+    validate( state.getContentWindows() );
+
+    if( state.getVersion() <  WINDOW_TITLES_VERSION )
+    {
+        state.getDisplayGroup().setX( displayGroup_->x());
+        state.getDisplayGroup().setY( displayGroup_->y());
+        state.getDisplayGroup().setWidth( displayGroup_->width());
+        state.getDisplayGroup().setHeight( displayGroup_->height());
+    }
+
+    *displayGroup_.get() = state.getDisplayGroup();
```

Comment:
remove .get(), the boost smart pointers can be dereferenced directly like normal pointers.